IntegrationTests/PurchaseTester/Podfile

```diff
@@ -1,18 +1,26 @@
-# Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
 
 target 'PurchaseTester' do
-
+  platform :ios, '12.1'
   # Pods for PurchaseTester
   pod 'Purchases', :path => '../../'
   pod 'PurchasesCoreSwift', :path => '../../'
 
 end
 
 target 'WatchPurchaseTester Extension' do
+  platform :watchos, '6.2'
 
   # Pods for PurchaseTester
   pod 'Purchases', :path => '../../'
   pod 'PurchasesCoreSwift', :path => '../../'
 
 end
+
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '12.0'
+    end
+  end
+end
```

Comment:
I missed this when reviewing, why are we setting it here? also, shouldn't this be the same value as defined in line 3?

Reply:
ooffff you're right, I missed this. I'll update it when I put my first PR up for PurchaseTest updates.